### PR TITLE
typo? “a substitution semantics”

### PR DESCRIPTION
**Pull request by cPlayIt**
missing word

### CODE REVIEW

**Comment by ggreif** [comment]
I think _substitution semantics_ is a thing. @crusso maybe add a (parenthesised) expansion?

**Comment by crusso** [comment]
You mean something like `(familiar from high school algebraic simplification)`?

**Comment by ggreif** [comment]
What about `(i.e. replacing equals with equals, as familiar from high school algebraic simplification)`

**Comment by cPlayIt** [comment]
... and I'm new to this process, am I supposed to modify it to your
suggestion or is replying to the email sufficient?

**Comment by ggreif** [comment]
@cPlayIt you are free to make suggestions either way. Or just have us a checkpoint committed and later improve on that :-)

File: doc/modules/language-guide/pages/basic-concepts.adoc [hunk 1]
```diff
@@ -121,7 +121,7 @@ x * y + x;
 
 === Declarations and variable substitution
 
-Declarations can be mutually recursive, but in cases where they are not, they permit a substitution semantics.
+Declarations can be mutually recursive, but in cases where they are not, they permit a substitution of semantics.
```
Comment (crusso) [inline doc/modules/language-guide/pages/basic-concepts.adoc:124]:
```suggestion
Declarations can be mutually recursive, but in cases where they are not, they permit a substitution semantics  
(that is, replacing equals for equals, as familiar from high-school algebraic simplification).
```

Comment (crusso) [reply doc/modules/language-guide/pages/basic-concepts.adoc:124]: @cPayIt, I've made the correction as a suggestion above. You can now commit the suggestion and ask us to review it again.

**Review by crusso — APPROVED** [review]
LGTM

**Pull request closed without merging**